buildenv/jenkins/common/test

```diff
@@ -118,6 +118,14 @@ def test_all() {
 def test_all_with_fetch() {
     fetch_artifacts()
     test_all()
-    cleanWs()
+
+    try {
+        cleanWs()
+    } catch(e) {
+        echo e.toString()
+        if (SPEC.startsWith == "win_") {
```

Comment:
Can you verify that this syntax will work?  It shouldn't be ```if(SPEC.startsWith("win_"))```?

Reply:
good catch, fixed